nasl/nasl_init.c

```diff
@@ -193,16 +193,24 @@ static init_func libfuncs[] = {
   {"sleep", nasl_sleep},
   {"isnull", nasl_isnull},
   {"defined_func", nasl_defined_func},
-  {"forge_ip_packet", forge_ip_packet},
+
+  /* Following 5 entries needed for backwars compatibility */
   {"forge_ipv6_packet", forge_ipv6_packet},
-  {"get_ip_element", get_ip_element},
   {"get_ipv6_element", get_ipv6_element},
-  {"set_ip_elements", set_ip_elements},
   {"set_ipv6_elements", set_ipv6_elements},
-  {"insert_ip_options", insert_ip_options},
   {"insert_ipv6_options", insert_ipv6_options},
-  {"dump_ip_packet", dump_ip_packet},
   {"dump_ipv6_packet", dump_ipv6_packet},
+
+  {"forge_ip_packet", forge_ip_packet},
+  {"forge_ip_v6_packet", forge_ipv6_packet},
```

Comment:
Any strong reasons against renaming the internal functions as well because we currently still have `forge_tcp_v6_packet` vs. `forge_ipv6_packet` for the internal functions.

Reply:
I checked if these functions are called internally and it does not seem like it. So yes, good point. The internal function names can be changed too.